presentations/index.md

```diff
@@ -1,10 +1,13 @@
+This is a collection of Gluster presentations from all over the world. 
+We have a [slideshare account](http://www.slideshare.net/GlusterCommunity) where most of these presentations are stored. 
+
```

Comment:
ndevos, any one can send a PR  with the url of their presentation. I am not sure how we can authorize the common slideshare account though.